starfish/pipeline/features/pixels/combine_adjacent_features.py

```diff
@@ -0,0 +1,117 @@
+from typing import List, Tuple, Union
+
+import numpy as np
+import pandas as pd
+from skimage.measure import regionprops, label
+
+from starfish.munge import dataframe_to_multiindex
+from starfish.constants import Indices
+from starfish.intensity_table import IntensityTable
+
+Number = Union[int, float]
+
+
+def combine_adjacent_features(
+        intensities: IntensityTable, area_threshold: Number, connectivity: int=2,
+) -> Tuple["IntensityTable", List, np.ndarray, np.ndarray]:
```

Comment:
Remove "s.

Reply:
Also, this still has too many return values.

Reply:
Oh, I see, you changed it in #337

Reply:
Ah, sorry. I should have changed it here. But yes, this is fixed.